rpc-server/src/dispatchers/consensus.rs

```diff
@@ -678,6 +678,57 @@ impl ConsensusInterface for ConsensusDispatcher {
         self.send_raw_transaction(raw_tx).await
```

Comment:
I would probably have just called `send_update_staker_transaction` here, but that's a minor thing as the method will be removed anyway.